.travis.yml

```diff
@@ -6,4 +6,6 @@ python:
 install:
   - "pip install ."
   - "pip install -r requirements.txt"
-script: py.test
\ No newline at end of file
+script:
+  - py.test /sqlalchemy_jsonapi/tests/
```

Comment:
Hm, this path looks fishy, and it doesn't look like travis has picked up on the tests. I think, at the very least, the path should start with `./`